libs/blocks/preflight/panels/general.js

```diff
@@ -110,6 +118,14 @@ function toggleGroup(name) {
   content.value = { ...content.value };
 }
 
+function checkPublishing(item, isFetching) {
+  if ((item.preview === DEF_NEVER && item.live === DEF_NEVER)
+    || (item.preview === DEF_NOT_FOUND && item.live === DEF_NOT_FOUND)) {
+    return ' not-found';
+  }
+  return isFetching;
+}
```

Comment:
Nit: this could be cleaner
```suggestion
function checkPublishing(item, isFetching) {
  return [DEF_NEVER, DEF_NOT_FOUND].some(status => item.preview === status && item.live === status)
    ? 'not-found' 
    : isFetching;
}
```